### DISCUSSION TITLE
How can I test a config change without merging it?

### DISCUSSION THREAD

**Discussion by allanlewis** [renovatebot/renovate#24180] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I want to see Renovate's explanation of the effect of a proposed config change.

### What you tried so far.

I'm raising PRs with config changes but I can't see a way to verify what effect they'll have without merging them and running Renovate again.

### Relevant debug logs

N/A

**Comment by rarkins** [comment, marked as answer]
The closest you can get is the platform=local option
